tools/Dockerfiles/ubuntu_jdk8

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:latest
```

Comment:
Do you want to have a ubuntu image pulled? Travis runs on Ubuntu 14.04 (stable) version. They usually run their infra on a LTS version. Why not use the Travis' underlying ubuntu directly?

Reply:
Because Ubuntu LTS 14.04 is 4 years out of date at this point. 16.04 and 18.04 are both LTS versions that are newer than 14.04. Also, 14.04.5 LTS EOLs on April 2019. `ubuntu:latest` tracks the latest LTS version of Ubuntu (18.04 -> 20.04 in 2 years, etc). 

This allows:
- Us to test locally w/o travis on a common platform (so if branch `xyz` fails this Ubuntu test on Travis, you can run with the exact same environment locally -- this is what containers gives us).
- Us to change the test (or add a new test) to support JDK 11 which won't ship on 14.04.
- Us to test other (fixed) versions of Ubuntu if we want. (18.10 when/before it ships, 19.04, etc. if we want to track testing versions of Ubuntu).

Reply:
(See https://hub.docker.com/_/ubuntu/ for more information)